Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -274,12 +277,16 @@ - (instancetype)_commonInitializationWithProcessPool:(WebKit::WebProcessPool&)pr
 - (void)_setupVisibilityPropagationViewForWebProcess
 {
 #if USE(EXTENSIONKIT)
-    if (!_visibilityPropagationInteractionForWebProcess) {
+    if (!_visibilityPropagationInteractionForWebProcess && !_visibilityPropagationViewForWebProcess) {
         SEL selector = NSSelectorFromString(@"createVisibilityPropagationInteraction");
         if ([_page->process().extensionProcess().get() respondsToSelector:selector])
             _visibilityPropagationInteractionForWebProcess = [_page->process().extensionProcess() performSelector:selector];
-        if (_visibilityPropagationInteractionForWebProcess)
-            [self addInteraction:_visibilityPropagationInteractionForWebProcess.get()];
+        if (_visibilityPropagationInteractionForWebProcess) {
+            _visibilityPropagationViewForWebProcess = adoptNS([[UIView alloc] initWithFrame:CGRectZero]);
```

Comment:
So these are "invisible" UIViews that the user cannot interact with, it seems.

Reply:
Exactly, their only purpose is to take care of the visibility propagation.